<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Destiny Passwords</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Password will appear here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter length</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
